doc/first-node.rst

```diff
@@ -207,90 +213,72 @@ First Deployment
    (on other microcontrollers make sure you have some button connected
    and know the corresponding GPIO port).
 
-2. In the ``test01`` (and inside the iot environment) folder,
-   run the following command
-
-   In Linux (if you run in a virtual machine, pass through your serial USB
-   of your microcontroller): ``deploy serial`` (or select from cloudcmd menu)
+2. In the ``test01`` folder (and inside the iot environment),
+   run the deployment command.
 
-   In Windows:
-
-   - In Windows using WSL 2 or in docker, you currently can only remote flash.
-     You can just run ``compile`` instead of ``deploy serial`` and then download
-     the firmware from your build folder and flash it manually.
+   **On Linux:**
    
-   - You can also setup net2ser on your router.
-     Run the following command, but replace the X with the number after COM (in the above example X = 8)
-
-     And, even more exciting, you can now flash devices (i.e. Wemos D1 Mini) 
```

Comment:
Add this detailed net2ser description into it's own documentation and refer to it from here and from quickstart